bin/glob

```diff
@@ -180,7 +180,7 @@ use File::Spec;
 
 use vars qw($VERSION @ISA @EXPORT @EXPORT_OK);
 
-$VERSION = 1.2_06;
+$VERSION = 1.2_07;
```

Comment:
We can probably make this a non-trial version unless you have a reason you'd like to keep it as such.